frontend/packages/shipwright-plugin/src/components/build-list/BuildTable.tsx

```diff
@@ -52,16 +60,51 @@ export const BuildHeader = () => {
       props: { className: columnClassNames[3] },
     },
     {
-      title: '',
+      title: t('shipwright-plugin~LastRun'),
```

Comment:
As discussed, also for the other columns, please use spaces like this:

```suggestion
      title: t('shipwright-plugin~Last run'),
```